saleor/static/dashboard/scss/document.scss

```diff
@@ -136,251 +28,148 @@ td, th {
   font-weight: 700;
 }
 
+*, :after, :before {
+  -webkit-box-sizing: inherit;
+  box-sizing: inherit;
+}
+
+html, body {
+  font-size: 12px;
+}
+
 html {
+  -ms-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
   line-height: 1.5;
   font-family: Roboto, sans-serif;
   font-weight: 400;
   color: rgba(0, 0, 0, 0.87);
 }
 
-h1, h2, h3, h4, h5, h6 {
-  font-weight: 400;
-  line-height: 1.1;
+body {
+  margin: 0;
 }
 
-h1 {
-  font-size: 3rem;
-  margin: 1.5rem 0 1.2rem;
-  line-height: 110%;
+details, header {
+  display: block;
+}
+
+header {
+  margin-bottom: 5rem;
 }
 
 h2 {
-  line-height: 110%;
   font-size: 2rem;
   margin: 1rem 0 0.8rem;
```

Comment:
You are using px in the entire file.

Reply:
I'm not sure if I get it right - are you telling me that I should stick to px and keep code consistency, or i should switch to rem?

Reply:
You should be consistent.